src/util/iterative_restoration.jl

```diff
@@ -224,7 +308,7 @@ function _build_rop_ir(pm::_PM.AbstractPowerModel)
         _PM.variable_shunt_admittance_factor(pm, nw=n, relax=true)
 
         constraint_restoration_cardinality_ub(pm, nw=n)
-        constraint_restoration_cardinality_lb(pm, nw=n)
+        # constraint_restoration_cardinality_lb(pm, nw=n)
```

Comment:
Now that the bug is figured out, can this be restored?

Reply:
This bug about damaged shunts ended up not being caused by the `cardinality_lb` constraint, but theoretically a network that requires multiple components to be restored at the same time would fail this program.  

Code to check if multiple items are restored simultaneously and add extra empty repair networks appropriately is also a part of this pull request. 

That said, no current tests fail if this constraint is re-added. Should a network that fails strict cardinality of restoration components be added?

Reply:
I got it.  Sounds good.  Maybe we should then remove the commented out line.

Reply:
done